activesupport/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 *   Add `String#upcase_first` method.
 
-     *Glauco Custódio*
+    *Glauco Custódio*, *bogdanvlviv*
```

Comment:
I did not give a new notice in CHANGELOG because of  method was never released, but a think i helped to create this method correctly. this method will be work correctly because of me.